llumnix/config/llumnix_config.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2024, Alibaba Group;
```

Comment:
config.py is enough?

Reply:
or cli_configs.py?

Reply:
Just keep lluminx_config.py for the following reasons:

1. We also have lluminx/internal_config.py to avoid any ambiguity.
2. This file contains the class LlumnixConfig, so naming the file lluminx_config.py is appropriate.

Reply:
1. We have many LlumnixXXX class, but none of these files called llumnix_xxx.py. 2. The parent directory already names Llumnix, and containing Llumnix in filename again is a bit duplicated.

Reply:
Indeed, it is a bit redundant. Change it to config.py